### ISSUE TITLE
Interest in an '%in%' function?

### ISSUE THREAD

**Issue by ChrisMuir** [nfultz/ht#1]
I was messing around with this package and created a simple `%in%` function for objects with class `ht`, looks like this:
```r
#' @export
#' @rdname ht
`%in.ht%` <- function(index, x) {
  !is.null(x[index]);
}
```
And works like this:
```r
library(ht)
x <- ht()
x[1:2] <- 3:4

1:2 %in.ht% x
#> TRUE

"cats" %in.ht% x
#> FALSE
```
If there's any interest in adding this function, I'd be happy to submit via PR.

**Comment by nfultz** [comment]
I would be generally open to this.

**Comment by nfultz** [comment]
Closed by #2
